runtime/legion/legion_tasks.cc

```diff
@@ -4203,6 +4305,22 @@ namespace Legion {
         deactivate();
     }
 
+    //--------------------------------------------------------------------------
+    void MultiTask::slice_index_space_for_replay(LegionTrace *trace)
```

Comment:
This is alright for now, but we need to document some way that for distributed tracing we're going to need to do something different as making one slice that handles all the points won't really be practical.